src/library/rekordbox/rekordboxfeature.cpp

```diff
@@ -1289,10 +1297,10 @@ QString RekordboxFeature::formatRootViewHtml() const {
             << tr("Beatgrids")
             << tr("Hot cues")
             << tr("Memory cues")
-            << tr("Loops (after first not usable in Mixxx 2.3 yet)");
+            << tr("Loops (not yet supported in Mixxx 2.3)");
```

Comment:
The old sentence was correct, although not proper English.

This imports loops, but Mixxx only supports using the first loop right now (it's the disabled, restored loop after track load). The other loops are currently not usable because #2194 has not been merged yet.